pkg/domain/peer.go

```diff
@@ -8,56 +8,43 @@ import (
 )
 
 type Peer struct {
-	Id    string
-	Host  string
-	Ports struct {
-		P2p string
-		Api string
-	}
-}
-
-func errInvalidAddr(addr string) error {
-	return fmt.Errorf("invalid addr format %s", addr)
+	Id     string
+	Host   netip.AddrPort
+	Active bool
```

Comment:
Mostly simplifying the data structure and adding `Active` which indicates if the peer should be included in the Quorum.